tests/emerge_new_or_changed_ebuilds.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
+#
```

Comment:
OK, I'd like to suggest to use google's shell script style/formatting guide for our shell scripts so we can ensure they are consistent.
See https://google.github.io/styleguide/shell.xml

In there they have the same format for the file header comment :)